manticore/core/cpu/abstractcpu.py

```diff
@@ -162,7 +165,184 @@ def __contains__(self, register):
 
         :param register: a register name
         '''
-        return self._alias(register) in self.all_registers 
+        return self._alias(register) in self.all_registers
+
+
+class ABI(object):
```

Comment:
pedantic naming thing but for class `Cpu`, we only capitalized the first letter, do the same here to be consistent?